src/components/RoleQuizModal/QuizStage.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useMemo, useState } from "react"
+import { Box } from "@chakra-ui/react"
+import QuizControls from "./QuizControls"
+import { H5 } from "../../components/Typography"
+import { GoBack, GoForward, QuizOption, QuizStageData, Result } from "./types"
+import QuizRadioGroup from "./QuizRadio"
+import DefiAccordion from "./DefiAccordion"
+
+const QuizStage: FC<{
+  stage: QuizStageData
+  goForward: GoForward
+  goBack: GoBack
+  displayBackButton: boolean
+}> = ({ stage, goBack, goForward, displayBackButton }) => {
+  const [value, setValue] = useState("")
+
+  const selectedOption: QuizOption = useMemo(
+    () => stage.options.filter((option) => option.label === value)[0],
+    [value]
+  )
+
+  const goForwardAndClearSelectedOption = (result: Result) => {
+    setValue("")
+    goForward(result)
+  }
+
+  return (
+    <Box>
+      <H5>{stage.title}</H5>
+      <QuizRadioGroup stage={stage} setValue={setValue} value={value} />
+      <QuizControls
+        displayBackButton={displayBackButton}
+        selectedOption={selectedOption}
+        goForward={goForwardAndClearSelectedOption}
+        goBack={goBack}
+      />
+      {stage.includeAccordion && <DefiAccordion />}
```

Comment:
![Screen Shot 2022-04-13 at 2 31 24 PM](https://user-images.githubusercontent.com/15269424/163265422-734830b5-15c4-4b22-9a1c-27fd1e8bb72e.png)

I think it is a bit more clear after adding the defi accordion content to the CMS. Let me know what you think.

Reply:
I don't think we need to list out accordion items for the CMS because this isn't even a "real" accordion like we have on the FAQ page. It's just some collapsed text with a bit of extra info.